changelogs/unreleased/7325-release-tool-version-tag.yml

```
@@ -0,0 +1,5 @@
+description: Remove the version_tag from the metadata of the .cfg files.
```

I would make this a bit more descriptive to an end user
```suggestion
description: "Module release tool: do not write internal field `version_tag` to `setup.cfg`.
```